pymc3/tests/test_sampling.py

```diff
@@ -180,13 +175,9 @@ def test_trace_report_bart(self):
         assert var_imp[0] > var_imp[1:].sum()
         npt.assert_almost_equal(var_imp.sum(), 1)
 
-    def test_return_inferencedata(self):
+    def test_return_inferencedata(self, monkeypatch):
```

Comment:
Who supplies the `monkeypatch` kwarg? Is that something that `pytest` does automatically?

Reply:
yes, it's a built-in pytest fixture https://docs.pytest.org/en/stable/monkeypatch.html